src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -130,6 +131,10 @@ const BoxAISidebar = (props: BoxAISidebarProps) => {
         }
     }, [questions.length, setHasQuestions]);
 
+    if (renderRemoteModule) {
+        return <>{renderRemoteModule(elementId)}</>;
```

Comment:
```suggestion
        return renderRemoteModule(elementId);
```

do we need to wrap it in a `div` if its returning a single `ReactNode`?

Reply:
Feel like we do not need to introduce extra `div` here, will just remove the `<>`